src/chum.cpp

```diff
@@ -60,6 +62,26 @@ void Chum::setShowAppsByDefault(bool v) {
   emit showAppsByDefaultChanged();
 }
 
+void Chum::setManualVersion(const QString &v) {
+    if (!m_ssu.manageRepo()) {
+        emit error(qtTrId("chum-repo-management-disabled-title"));
+        return;
+    }
+
+    if (m_manualVersion == v) return;
+    m_manualVersion = v;
+
+    QSettings settings;
+    settings.setValue(s_config_manualversion, v);
+    emit manualVersionChanged();
+
+    m_busy = true;
+    emit busyChanged();
+    //% "Setting up Chum repository"
```

Comment:
As this string is defined elsewhere in the code, we actually don't need string here at all. I suggest to delete that line. Probably I should check my own code for such duplication as well...